packages/jaeger-ui/src/utils/readJsonFile.tsx

```diff
@@ -22,8 +22,10 @@ export default function readJsonFile(fileList: { file: File }) {
       }
       try {
         resolve(JSON.parse(reader.result));
-      } catch (error: any) {
-        reject(new Error(`Error parsing JSON: ${error.message}`));
+      } catch (error: unknown) {
+        if (error instanceof Error) {
+          reject(new Error(`Error parsing JSON: ${error.message}`));
+        }
```

Comment:
how about this then? If the type is already Error, its toString will return the message, but if it's some other object not extending error, then we get `[Object object]`. But at least in all cases we do call `reject`, which your code may not call.
```suggestion
      } catch (error: object) {
          reject(new Error(`Error parsing JSON: ${error}`));
      }
```

Reply:
Ok, but we cannot use `error: object` as well it gives same error that I mention above. So, in this case we have revert it back to any.

Reply:
Ok then go with any

Reply:
And a lint ignore line